operations/_scripts/generate/generate_identifier.sh

```diff
@@ -17,4 +17,5 @@ if [ -z "$AWS_RESOURCE_IDENTIFIER" ]; then
 else
   GITHUB_IDENTIFIER="$AWS_RESOURCE_IDENTIFIER"
 fi
+# todo: echo "$GITHUB_IDENTIFIER" | xargs | tr '[:upper:]' '[:lower:]' | tr '_' '-' | tr '/' '-' 
```

Comment:
What's going on here?

Reply:
Maybe [this](https://github.com/bitovi/github-actions-deploy-docker-to-ec2/issues/105) should be here?

Will do so in next PR, want to get this done before doing cosmetics